src/config.rs

```diff
@@ -41,6 +41,10 @@ pub struct Config {
     // Gitlab authentication
     pub(crate) gitlab_accesstoken: Option<String>,
 
+    // Access token for rebuild trigger at path
+    // "/crate/:name/:version/rebuild"
+    pub(crate) trigger_rebuild_token: Option<String>,
```

Comment:
I'm not sure about naming here, if that token will only be used for this API, or all future docs.rs APIs for crates.io. 

I assume the latter